src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformer.java

```diff
@@ -212,17 +214,22 @@ private PackageInfoView generatePackageInfo(
     String outputPath = apiConfig.getPackageName();
     String fileName = "doc.go";
     String doc = model.getServiceConfig().getDocumentation().getSummary();
-    return PackageInfoView.newBuilder()
-        .templateFileName(DOC_TEMPLATE_FILENAME)
-        .outputPath(outputPath + File.separator + fileName)
-        .serviceTitle(model.getServiceConfig().getTitle())
-        .importPath(apiConfig.getPackageName())
-        .packageName(namer.getLocalPackageName())
-        .serviceDocs(Collections.<ServiceDocView>emptyList())
-        // TODO(pongad): GoContextCommon should be deleted after we convert Go discovery to MVMM.
-        .packageDoc(new GoContextCommon().getWrappedCommentLines(doc))
-        .domainLayerLocation(apiConfig.getDomainLayerLocation())
-        .build();
+    PackageInfoView.Builder packageInfo = PackageInfoView.newBuilder();
+
+    packageInfo.templateFileName(DOC_TEMPLATE_FILENAME);
```

Comment:
Nothing wrong with this, but wondering why you're against using the chained calls?